ios/native/NativeMethods.mm

```diff
@@ -42,6 +43,21 @@ void scrollTo(int scrollViewTag, RCTUIManager *uiManager, double x, double y, bo
   [scrollView scrollToOffset:(CGPoint){(CGFloat)x, (CGFloat)y} animated:animated];
 }
 
+void dispatchCommand(RCTUIManager *uiManager, NSNumber *viewTag, NSString *commandID, NSArray *commandArgs)
+{
+  SEL privateMethodSelector = NSSelectorFromString(@"dispatchViewManagerCommand:commandID:commandArgs:");
+  NSInvocation *invocation =
+      [NSInvocation invocationWithMethodSignature:[uiManager methodSignatureForSelector:privateMethodSelector]];
+  [invocation setSelector:privateMethodSelector];
+  [invocation setTarget:uiManager];
+  [invocation setArgument:&viewTag atIndex:2];
+  [invocation setArgument:&commandID atIndex:3];
+  [invocation setArgument:&commandArgs atIndex:4];
+  RCTUnsafeExecuteOnUIManagerQueueSync(^{
+    [invocation invoke];
+  });
+}
```

Comment:
Maybe we could extend category instead of this like here - https://github.com/software-mansion/react-native-reanimated/blob/main/ios/native/NativeProxy.mm#L35-L37